build-configs/malte/index.ts

```diff
@@ -19,6 +19,7 @@ const commonMalteBuildConfig: CommonBuildConfigType = {
   appIcon: 'app_icon_malte',
   notificationIcon: 'notification_icon_malte',
   lightTheme,
+  highContrastTheme: lightTheme,
```

Comment:
:question: shouldn't it be highContrastTheme here ?

Reply:
This will be done in upcoming tickets where I will implement the functional part :)